engine/table/src/main/java/io/deephaven/engine/tablelogger/ServerStateLogLogger.java

```diff
@@ -1,6 +1,7 @@
 package io.deephaven.engine.tablelogger;
 
 import io.deephaven.tablelogger.Row;
+import io.deephaven.tablelogger.Row.Flags;
 
 import java.io.IOException;
 
```

Comment:
Would be lovely to rename the default flags. Shouldn't mention "intraday logger".